recipes/xmlsec/all/conanfile.py

```diff
@@ -0,0 +1,213 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment, VisualStudioBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+from contextlib import contextmanager
+import os
+
+class XmlSecConan(ConanFile):
+    name = "xmlsec"
+    description = "XML Security Library is a C library based on LibXML2. The library supports major XML security standards."
+    license = ("MIT", "MPL-1.1")
```

Comment:
In the repo I dont see any Mozilla license, can you confirm where it's listed?

Reply:
Saw it in his Copyright file:
https://github.com/lsh123/xmlsec/blob/master/Copyright#L43-L47

Reply:
Perfect that you! I missed that (looking for the whole thing)